.github/workflows/release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           echo ossrhUsername=eller86 >> gradle.properties
           echo "ossrhPassword=${SONATYPE_PASSWORD}" >> gradle.properties
           echo "keystorepass=${KEYSTORE_PASS}" >> gradle.properties
-          ./gradlew publish --no-daemon
+          ./gradlew assemble publish --no-daemon --parallel
```

Comment:
Doesn't `publish` imply `assemble`? No biggie :)

Reply:
Yes, Eclipse plugin and site aren't targeted to `publish`, so we need to invoke `assemble` explicitly.

Reply:
Nice catch :)